packages/yoco/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
did lerna add this? :D

Reply:
I thought it’s a fuck-up by me, but actually it turns out to be [from the original source code](https://github.com/3YOURMIND/yoco/blob/5a40fbf4d3f9915e66b174fbe0d8f6ca082524a0/yarn.lock). This can safely be deleted though.